services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -9,6 +9,31 @@
 from types_aiobotocore_ec2.literals import InstanceStateNameType, InstanceTypeType
 
 
+class SimcoreServiceDockerLabelKeys(BaseModel):
+    # NOTE: in a next PR, this should be moved to packages models-library and used
+    # all over, and aliases should use io.simcore.service.*
+    # https://github.com/ITISFoundation/osparc-simcore/issues/3638
```

Comment:
NOTE: we have this functionality as well in the service integration library . There I think is pretty solid. I agree that we should somehow put all this code in a single place. I will add some hints in the issue above